.github/workflows/coverage.yaml

```diff
@@ -0,0 +1,32 @@
+on: [push]
+
+name: Code Coverage
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - uses: actions-rs/toolchain@v1
+        with:
+          toolchain: nightly
+          override: true
+      - uses: actions-rs/cargo@v1
+        with:
+          command: test
+          args: --all-features --no-fail-fast
+        env:
+          CARGO_INCREMENTAL: '0'
+          RUSTFLAGS: '-Zprofile -Ccodegen-units=1 -Cinline-threshold=0 -Clink-dead-code -Coverflow-checks=off -Zno-landing-pads'
```

Comment:
I understand that `-Coverflow-checks=off` disables checking for integer overflows.  Why do you want to disable it?

Reply:
I see that there's another action that does the `cargo test` without flags in ` .github/workflows/rust.yaml`.

Reply:
I kept this default configuration since I was interested only in code coverage (I took it from the codecov.io site, IIRC).

I just realized that the other cargo test also needs `--all-features` :/